x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/components/tables/engines_table.tsx

```diff
@@ -50,7 +52,7 @@ export const EnginesListTable: React.FC<EnginesListTableProps> = ({
         defaultMessage: 'Documents',
       }),
       dataType: 'number',
-      render: (number: number) => <FormattedNumber value={number} />,
+      render: () => <FormattedNumber value={0} />,
```

Comment:
Backend does not include` documents count` in the response. Replaced render to show default value `0`

Reference - [comment](https://github.com/elastic/kibana/pull/148352#discussion_r1060997860)

Reply:
I think we can remove this whole column now given the feedback from Nick in the engines sync.